app/src/main/java/com/android/sample/ui/activity/EditActivity.kt

```diff
@@ -151,9 +168,75 @@ fun EditActivityScreen(
+                ) {
+                  Row(
+                      horizontalArrangement =
+                          Arrangement.spacedBy(8.dp, Alignment.CenterHorizontally),
+                      verticalAlignment = Alignment.CenterVertically,
+                  ) {
+                    Icon(
+                        Icons.Filled.Add,
+                        contentDescription = "add a new attendee",
+                    )
+                    Text("Add Attendee")
+                  }
+                }
+
+                LazyRow(
+                    modifier = Modifier.fillMaxHeight().height(85.dp).padding(8.dp),
+                ) {
+                  items(attendees.size) { index ->
+                    Row(
+                        modifier = Modifier.padding(8.dp).background(Color(0xFFFFFFFF)),
+                    ) {
+                      Text(
+                          text = attendees[index].name,
+                          modifier = Modifier.padding(8.dp),
+                          style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+                      )
+                      Spacer(modifier = Modifier.width(8.dp))
+                      Text(
+                          text = attendees[index].surname,
+                          modifier = Modifier.padding(8.dp),
+                          style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+                      )
+                      Spacer(modifier = Modifier.width(8.dp))
+                      Text(
+                          text = attendees[index].age.toString(),
+                          modifier = Modifier.padding(8.dp),
+                          style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+                      )
+                      Button(
+                          onClick = { attendees = attendees.filter { it != attendees[index] } },
+                          modifier =
+                              Modifier.width(40.dp).height(40.dp).testTag("removeAttendeeButton"),
+                      ) {
+                        Icon(
+                            Icons.Filled.PersonRemove,
+                            contentDescription = "remove attendee",
+                        )
+                      }
+                    }
+                  }
+                }
+              }
+          if (showDialog) {
+            AddUserDialog(
+                onDismiss = { showDialog = false },
+                onAddUser = { user -> attendees = attendees + user })
+          }
           Spacer(modifier = Modifier.height(32.dp))
           Button(
-              enabled = title!!.isNotEmpty() && description!!.isNotEmpty() && dueDate.isNotEmpty(),
+              enabled =
+                  title?.isNotEmpty() ?: false &&
+                      description?.isNotEmpty() ?: false &&
+                      dueDate.isNotEmpty(),
```

Comment:
try using this : 
```
enabled = title.orEmpty().isNotEmpty() &&
          description.orEmpty().isNotEmpty() &&
          dueDate.isNotEmpty()
```


it manages the null cases with a better readability

Reply:
but why would this be null ? don't you already manage this when creating the variable ?